universitiesPage/universities.html

```diff
@@ -20,6 +20,16 @@
             </div>
 
             <div>
+
+                <ul id="list">
+                    <li><a href="../index.html">home</a></li>
+                    <li id="search"><a href="search.html">search</a></li>
+                    <li><a href="../aboutPage/about.html">about us</a></li>
+                </ul>
+    
+            </div>
+
+            <div id="menu">
```

Comment:
The button in the Universities page takes a different design
check it, please!

Reply:
I already modified the design on the university page, check it when I upload the responsive design!